lib/index.js

```diff
@@ -68,6 +68,11 @@ async function replace( streamObj, replacements, binary = null ) {
 	}
 	debug( 'The supplied arguments are valid' );
 
+	if ( ! replacements.length ) {
+		console.log( 'No replacements were provided to search and replace with.' );
```

Comment:
Should this use `debug` rather than writing to stdOut?

```suggestion
		debug( 'No replacements were provided to search and replace with.' );
```

Reply:
Good call 👍